Extensions/IISWebAppDeploy/Tests/Tasks/IISWebAppMgmt/AppCmdOnTargetMachines.Tests.ps1

```diff
@@ -303,7 +303,7 @@ Describe "Tests for verifying Add-SslCert functionality" {
         Mock Run-Command -Verifiable {return "", "" , "" , "", "HostName:port                : localhost:80", "Certificate Hash             : SampleHash"} -ParameterFilter { $failOnErr -eq $false }
 
         $output = Add-SslCert -port "80" -certhash "SampleHash" -hostname "localhost" -sni "true" -iisVersion "8.0" 4>&1 | Out-String
-
+        
```

Comment:
addressed in commit [9e81d78](https://github.com/mmikkelson/vsts-rm-extensions/commit/9e81d78d3efd94c6234d768a897b2c28bf60080d)